src/main/java/org/terasology/structureTemplates/internal/systems/ProtectedRegionClientSystem.java

```diff
@@ -58,7 +58,8 @@ private boolean isInProtectedRegion(Collection<Vector3i> positions) {
 
     @ReceiveEvent(priority = EventPriority.PRIORITY_CRITICAL, components = {NoInteractionWhenProtected.class})
     public void onActivationPredicted(ActivationPredicted event, EntityRef target) {
-        Vector3f position = event.getTarget().getComponent(LocationComponent.class).getWorldPosition();
+        Vector3f position = event.getTarget().getComponent(LocationComponent.class).getWorldPosition(new Vector3f());
+        //TODO: is this equivalent to  new Vector3i(position, RoundingMode.HALF_UP);
```

Comment:
@pollend I think we should change this in the future to use the conversion with rounding mode. Leaving this as is for now to be on the safe side.